include/seqan/arg_parse/arg_parse_version_check.h

```diff
@@ -108,7 +108,8 @@ struct VersionCheck
     std::string _version = "0.0.0";
     std::string _program;
     std::string _command;
-    std::string _path = _getPath();
+    const std::string _path = _getPath();
```

Comment:
no sorry. I wondered why I did not declare this const and forgot to change it back.